### PR TITLE
[Improvement-16534][Master] Switch task support includes method

### LINKED ISSUE #16534: [Improvement][Master] switch task doesn't support includes function
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I want to execute condition like `['abc','def'].includes(${output})` in switch task, but error occurs:
```
javax.script.ScriptException: TypeError: ["abc", "efg"].includes is not a function in <eval> at line number 1
	at jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:470)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:454)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:406)
	at jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:402)
	at jdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:155)
	at javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
	at org.apache.dolphinscheduler.server.master.utils.SwitchTaskUtils.evaluate(SwitchTaskUtils.java:35)
```
Actually the includes function can be supported by using polyfills as https://stackoverflow.com/questions/47543566/scriptengine-javascript-doesnt-support-includes mentioned.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ChaoquanTao**
## Purpose of the pull request

This PR add polyfill for js engine to support includes method. fix #16534

## Brief change log

- *modify SwtichTaskUtils to support includes method*

## Verify this pull request

This change added tests and can be verified as follows:

- *Added SwitchTaskUtilsTest#testIncludes to verify the change.*

(or)

### CODE REVIEW

**Review by SbloodyS — COMMENTED** [review]
Please run `mvn spotless:apply` to format the code.

File: dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/utils/SwitchTaskUtils.java [hunk 1]
```diff
@@ -40,9 +40,16 @@ public class SwitchTaskUtils {
 
     private static final NashornSandbox sandbox;
     private static final String rgex = "['\"]*\\$\\{(.*?)\\}['\"]*";
+    public static final String NASHORN_POLYFILL_ARRAY_PROTOTYPE_INCLUDES =
+            "if (!Array.prototype.includes) { Object.defineProperty(Array.prototype, 'includes', { value: function(valueToFind, fromIndex) { if (this == null) { throw new TypeError('\"this\" is null or not defined'); } var o = Object(this); var len = o.length >>> 0; if (len === 0) { return false; } var n = fromIndex | 0; var k = Math.max(n >= 0 ? n : len - Math.abs(n), 0); function sameValueZero(x, y) { return x === y || (typeof x === 'number' && typeof y === 'number' && isNaN(x) && isNaN(y)); } while (k < len) { if (sameValueZero(o[k], valueToFind)) { return true; } k++; } return false; } }); }";
```
Comment (Gallardot) [inline dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/utils/SwitchTaskUtils.java]: Can we format this code to make it easier to read and understand?

Comment (ChaoquanTao) [reply dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/utils/SwitchTaskUtils.java]: done

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by Gallardot — APPROVED** [review]

**Pull request merged**